remotes/docker/schema1/converter.go

```diff
@@ -103,8 +103,38 @@ func (c *Converter) Handle(ctx context.Context, desc ocispec.Descriptor) ([]ocis
 	}
 }
 
+// ConvertOptions provides options on converting a docker manifest.
+type ConvertOptions struct {
+	ManifestMediaType string
+
+	ConfigMediaType string
+}
+
+// ConvertOpt allows configuring a convert operation.
+type ConvertOpt func(context.Context, *ConvertOptions) error
+
+// ConvertToDockerSchema2Manifest is used to indicate that a schema1 manifest
+// should be converted into the media types for a docker schema2 manifest.
+func ConvertToDockerSchema2Manifest() ConvertOpt {
```

Comment:
Make an OCI version of this?

Reply:
The default is already OCI, do we want one regardless?

Reply:
I don't think we need another function. Not sure if we have a consistent way to name these options

Reply:
`UseDockerSchema2` maybe?